lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -3816,12 +3813,9 @@ void ByteCodeGenerator::StartEmitFunction(ParseNode *pnodeFnc)
 
         PushScope(paramScope);
 
-        if (funcInfo->byteCodeFunction->IsFunctionParsed() && funcInfo->GetParsedFunctionBody()->GetByteCode() == nullptr)
```

Comment:
>funcInfo->byteCodeFunction->IsFunctionParsed() && funcInfo->GetParsedFunctionBody()->GetByteCode() == nullptr [](start = 12, length = 109)

just for future note : we should have better state machine enums, to illustrate, deferred, undeferred, redeffered, neverdeffered. something along the line. it will make the debugging easier.